SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.Windows.Shared/ErrorAttachmentLog.cs

```diff
@@ -89,5 +89,14 @@ public override void Validate()
                 throw new ValidationException(ValidationException.Rule.CannotBeNull, "Data");
             }
         }
+
+        public bool ValidatePropertiesForAttachment()
```

Comment:
Should we do this in `Validate`?

Reply:
I think Validate will do too much validation for the point at which this method gets called.

Reply:
But I agree maybe it should be called in validate?

Reply:
@jaelim-ms Nope. It was the problem.

We have similar way in iOS (separate validate function)